File: CodeFreak1/ClientApp/src/app/Security/Dtos/roles-view-model.ts
import { RequestStatus } from "./request-status";

export class RolesViewModel extends RequestStatus {
  public RoleId: number;
  public Name: string;
  public Description: string;
  public CreatedOn: Date;
  public ModifiedOn: Date;
  public IsActive: boolean;
  public ModifiedBy: string;
  public CreatedBy: string;
}

File: CodeFreak1/ViewModel/RolesViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class RolesViewModel : RequestStatus
    {
        public int RoleId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool? IsActive { get; set; }
        public Guid? ModifiedBy { get; set; }
        public Guid? CreatedBy { get; set; }
    }
}
